modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -361,6 +363,383 @@ Additionally, the user requires some `ReadOnly` permissions, and some of the rol
 
 For more information about creating an account with only the required privileges, see link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-5372F580-5C23-4E9C-8A4E-EF1B4DD9033E.html[vSphere Permissions and User Management Tasks] in the vSphere documentation.
 
+
+[discrete]
+[id="installation-vsphere-installer-infra--minimum-requirements_{context}"]
+== Minimum required vCenter account privileges
+
+After you create a custom role and assign privileges to it, you can create permissions by selecting specific vSphere objects and then assigning the custom role to a user or group for each object. 
+
+Before you create permissions or request for the creation of permissions for a vSphere object, determine what minimum permissions apply to the vSphere object. By doing this task, you can ensure a basic interaction exists between a vSphere object and {product-title} architecture. 
+
+[IMPORTANT]
+====
+If you create a custom role and you do not assign privileges to it, the vSphere Server by default assigns a `Read Only` role to the custom role. Note that for the cloud provider API, the custom role only needs to inherit the privileges of the `Read Only` role.
+====
+
+Consider creating a custom role when an account with global administrative privileges does not meet your needs. Also consider that depending on the installation stage, you might need to restrict access to certain vSphere objects.
+
+[IMPORTANT]
+====
+Installing an {product-title} cluster in a vCenter is tested against a full list of privileges as described in the "Required vCenter account privileges" section. By adhering to the full list of privileges, you can reduce the possibility of unexpected behaviors that might occur when creating a custom role with a restricted set of privileges. Red Hat does not support an environment that is configured to meet a minimum set of permissions. 
```

Comment:
Installing an {product-title} cluster in a vCenter is tested against a full list of privileges as described in the "Required vCenter account privileges" section. By adhering to the full list of privileges, you can reduce the possibility of unexpected behaviors that might occur when creating a custom role with a restricted set of privileges. Red Hat does not support an environment that is configured to meet a minimum set of permissions. 

maybe reword `Red Hat does not support an environment that is configured to meet a minimum set of permissions. ` to `Only an account configured with the required privileges is supported. `

Reply:
Thanks. Updated.

Reply:
That sounds reasonable.